bool isParking

float32 width
float32 depth

float32 top_y
float32 bottom_y
float32 out_x
float32 in_x